.github/workflows/build.yml

```diff
@@ -538,13 +539,14 @@ jobs:
             python:p
             shaderc:p
             spirv-cross:p
+            uchardet:p
+            vapoursynth:p
             vulkan-devel:p
 
       - name: Install dependencies
         if: ${{ matrix.sys != 'clang32' && matrix.sys != 'mingw32' }}
```

Comment:
should we get rid of all "clang32" references then?

Reply:
we can

Reply:
fixed

Reply:
Should be gone in master already? Unless I missed a line somewhere.

Reply:
Yeah, this patch was not rebased on mater, but there was another one in `ci/build-msys2.sh` that we missed.